src/share/poudriere/image.sh

```diff
@@ -476,14 +485,35 @@ zsnapshot)
 	;;
 esac
 
-# Use of tar given cpdup has a pretty useless -X option for this case
-tar -C ${mnt} -X ${excludelist} -cf - . | tar -xf - -C ${WRKDIR}/world
-touch ${WRKDIR}/src.conf
-[ ! -f ${POUDRIERED}/src.conf ] || cat ${POUDRIERED}/src.conf > ${WRKDIR}/src.conf
-[ ! -f ${POUDRIERED}/${JAILNAME}-src.conf ] || cat ${POUDRIERED}/${JAILNAME}-src.conf >> ${WRKDIR}/src.conf
-[ ! -f ${POUDRIERED}/image-${JAILNAME}-src.conf ] || cat ${POUDRIERED}/image-${JAILNAME}-src.conf >> ${WRKDIR}/src.conf
-[ ! -f ${POUDRIERED}/image-${JAILNAME}-${SETNAME}-src.conf ] || cat ${POUDRIERED}/image-${JAILNAME}-${SETNAME}-src.conf >> ${WRKDIR}/src.conf
-make -C ${mnt}/usr/src DESTDIR=${WRKDIR}/world BATCH_DELETE_OLD_FILES=yes SRCCONF=${WRKDIR}/src.conf delete-old delete-old-libs
+
+if [ -f "${PKGBASELIST}" ]; then
+	msg "Building pkgbase"
+	msg "mnt: ${mnt}"
+	OSVERSION=$(awk -F '"' '/REVISION=/ { print $2 }' ${mnt}/usr/src/sys/conf/newvers.sh | cut -d '.' -f 1)
+	msg "osversion: ${OSVERSION}"
+	mkdir -p ${WRKDIR}/world/etc/pkg/
+	cat << EOF > ${WRKDIR}/world/etc/pkg/FreeBSD-base.conf
+local: {
+        url: file://${POUDRIERE_DATA}/images/${JAILNAME}-repo/FreeBSD:${OSVERSION}:${arch}/latest,
+        enabled: true
+}
+EOF
```

Comment:
You can do `-EOF` to still use tabs on all of these lines and the tabs will be eaten.